packages/server/tests/acceptance.spec.ts

```diff
@@ -104,6 +103,48 @@ describe('RPC Server Integration Tests', async function () {
     this.timeout(180 * 1000);
 
     before(async function () {
+
+        logger.info(`Setting up Mirror Node Client for ${process.env['MIRROR_NODE_URL']} env`);
+        const mirrorNodeClient = Axios.create({
+            baseURL: `${process.env['MIRROR_NODE_URL']}/api/v1`,
+            responseType: 'json' as const,
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            method: 'GET',
+            timeout: 5 * 1000
+        });
+
+        // allow retries given mirror node waits for consensus, record stream serialization, export and import before parsing and exposing
+        axiosRetry(mirrorNodeClient, {
+            retries: 5,
+            retryDelay: (retryCount) => {
+                logger.info(`Retry delay ${retryCount * 1000} s`);
+                return retryCount * 1000;
+            },
+            retryCondition: (error) => {
+                // if retry condition is not specified, by default idempotent requests are retried
+                return error.response.status === 400 || error.response.status === 404;
+            }
+        });
+
+        const relayClient = Axios.create({
```

Comment:
If we are using / will be using `ethers.provider` we do not need this `relayClient` at all.

Reply:
Synced offline.